index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div">
+        <img id="image" src="https://upload.wikimedia.org/wikipedia/commons/b/ba/Norman_Borlaug.jpg" alt="">
+        <h2 id="img-caption">Dr. Borlaug giving a conference</h2>
+    </div>
+    <p id="tribute-info">
+      Norman Ernest Borlaug (Cresco, Iowa, Estados Unidos, 25 de marzo de 1914-Dallas, Texas, Estados Unidos, 12 de septiembre de 2009), citado como Norman Burlaug, fue un ingeniero forestal, genetista, fitopatólogo, humanista estadounidense, considerado por muchos el padre de la agricultura moderna y de la revolución verde, ha sido llamado "el hombre que salvó mil millones de vidas". Sus esfuerzos en los años 1960 para introducir las semillas híbridas a la producción agrícola en México, Pakistán e India provocaron un incremento notable de la productividad agrícola, y algunos lo consideran responsable de haber salvado más de 1000 millones de vidas humanas.
+    </p>
+    <a id="tribute-link" href="https://es.wikipedia.org/wiki/Norman_Borlaug" target="_blank">more info...</a>
+    </main>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Please do not leave outdated information as a comment in your code base. If you are gonna leave a comment, it must have a purpose and be used for information clarity. Kindly remove all outdated comments. 😃